src/24-hour-rule.ts

```diff
@@ -27,7 +28,7 @@ export const getMinimumOpenTime = (
 
   if (hasLabel(SEMVER_LABELS.MAJOR)) return MINIMUM_MAJOR_OPEN_TIME;
   if (hasLabel(SEMVER_LABELS.MINOR)) return MINIMUM_MINOR_OPEN_TIME;
-  if (hasLabel(SEMVER_LABELS.PATCH)) return MINIMUM_PATCH_OPEN_TIME;
+  if (hasLabel(SEMVER_LABELS.PATCH) || hasLabel(SEMVER_NONE_LABEL)) return MINIMUM_PATCH_OPEN_TIME;
```

Comment:
why `SEMVER_NONE_LABEL` and not `SEMVER_LABELS.NONE`?

Reply:
originally motivated by https://github.com/electron/cation/pull/51#issue-534677258